packages/stark-ui/src/modules/app-data/components/app-data.component.spec.ts

```diff
@@ -0,0 +1,129 @@
+/* tslint:disable:completed-docs*/
+import { StarkAppDataComponent } from "./app-data.component";
+import { STARK_LOGGING_SERVICE } from "@nationalbankbelgium/stark-core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { MatButtonModule } from "@angular/material/button";
+import { MatIconModule } from "@angular/material/icon";
+import { MatMenuModule } from "@angular/material/menu";
+import { CommonModule } from "@angular/common";
+import { TranslateModule, TranslateService } from "@ngx-translate/core";
+import { MatTooltipModule } from "@angular/material/tooltip";
+import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing";
+import { NO_ERRORS_SCHEMA } from "@angular/core";
+import { StarkSvgViewBoxModule } from "@nationalbankbelgium/stark-ui";
+
+//tslint:disable:no-big-function no-identical-functions
+describe("AppDataComponent", () => {
+	let component: StarkAppDataComponent;
+	let fixture: ComponentFixture<StarkAppDataComponent>;
+
+	const mockLogger: MockStarkLoggingService = new MockStarkLoggingService();
+
+	beforeEach(async(() => {
+		return TestBed.configureTestingModule({
+			declarations: [StarkAppDataComponent],
+			imports: [
+				CommonModule,
+				StarkSvgViewBoxModule,
+				MatButtonModule,
+				MatIconModule,
+				MatMenuModule,
+				MatTooltipModule,
+				TranslateModule.forRoot()
+			],
+			providers: [
+				{ provide: STARK_LOGGING_SERVICE, useValue: mockLogger },
+				TranslateService
+			],
+			schemas: [NO_ERRORS_SCHEMA] // tells the Angular compiler to ignore unrecognized elements and attributes (svgIcon)
```

Comment:
Do we really need this? Cause I see that the StarkSvgViewModule is in the "imports" of this testing module